app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -66,6 +68,13 @@ fun getBranchToDownloadAssetsFrom(assetType: String): String {
     }
 }
 
+fun getAvailableStorageInMB(): Long {
```

Comment:
AFAIK, there's no way to mock a top-level function like this, which I would guess is what you were talking about having trouble testing. Try inserting it into a class instead. It would be _kind of_ stretching the defintion of the filesystem utility, but you could add it there if you wanted, especially if you rephrased it a bit to something like `FilesystemUtility#spaceRequiredForNewFilesystemIsAvailable`. I can describe this more offline but I'm tired of typing.